diff --git a/templates/pages-template-worker.ts b/templates/pages-template-worker.ts
index 67e6cee9ac7cbac5bf88f8dc75f70be3a010c4fb..a67a6e3f6d19a370a5037141d50828f8269ad505 100644
--- a/templates/pages-template-worker.ts
+++ b/templates/pages-template-worker.ts
@@ -137,7 +137,7 @@ export default {
 			if (result.done === false) {
 				const { handler, params, path } = result.value;
 				const context = {
-					request: new Request(request.clone()),
+					request: new Request(request),
 					functionPath: path,
 					next,
 					params,
